web/components/atoms/ProgressBar.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props {
+	color?: string;
+	/** A number between 0 and 100 indicating the progress percentage */
+	progress: number;
+}
+
+export default function ProgressBar({
+	color,
+	progress
+}: Props) {
+	return (
+		<div className="h-[6px] relative rounded-full">
+			<div style={{ backgroundColor: color, width: `${progress}%` }} className="rounded-full absolute top-0 left-0 bottom-0 transition-all" />
```

Comment:
I like how you made color a variable in case we want to have different colors in the future, but I think it'll only be MBB's pink, so you can statically set this color as `"bg-mbb-pink"` in Tailwind since we define "mbb-pink" in `web/tailwind.config.js`!